src/components/dev-dot-content/mdx-components/mdx-heading-permalink/index.tsx

```diff
@@ -0,0 +1,15 @@
+import classNames from 'classnames'
+import { IconLink16 } from '@hashicorp/flight-icons/svg-react/link-16'
+import s from './mdx-heading-permalink.module.css'
+
+export default function MdxHeadingPermalink(props) {
+	const { className, level, ...rest } = props
+
+	return (
+		<a className={classNames(s.root, className, s[`h${level}`])} {...rest}>
+			<div className={s.icon}>
+				<IconLink16 color="var(--token-color-foreground-action)" />
+			</div>
```

Comment:
Is the wrapping `<div>` needed since we can pass a `className` directly to `IconLink16`? I think we also want to not se `color` here in the JSX, but in the CSS. Is there some further context behind using the `color` prop on `IconLink16`?

Reply:
I just put the color prop in there because that is what the default color is supposed to be before any action is taken place but we can move it to the css for more consistency. Also, we don't need that `<div>` any more. We can just add `display: flex;` on a class directly on the icon. Good call!